src/components/input-field/input-field.tsx

```diff
@@ -443,11 +445,27 @@ export class InputField {
             return false;
         }
 
+        if (this.type === 'url' && this.isUrlInvalid(this.value)) {
+            return true;
+        }
+
         const element = this.getInputElement();
 
         return !(element && element.checkValidity());
     }
 
+    private isUrlInvalid(url: string) {
```

Comment:
I'm not doing a full review of this, just leaving some thoughts here :slightly_smiling_face: 

It feels a bit unfortunate that we have to remove the native validation of URL fields and that this now does not have the same functionality as before, only because we are having issues with it in one place in the webclient. Perhaps it would be possible to solve the underlying problem behind this issue from crm-components instead by always passing `invalid = false`, basically turning off validation for the problematic fields? :thinking:

Reply:
We could also add a new `type`, e.g. `lime-url`, `relative-url` or something like that :thinking:

Reply:
I don't think we can, firstly because I don't think always passing in `false` will make any difference (we will treat the field as invalid if _either_ the input or the native check says it's invalid), and secondly because I already tried using `<input type="url">` and doing the following on lines 448-450:
```
        if (this.type === 'url') {
            return this.isUrlInvalid(this.value);
        }
```
That completely bypasses the native validation, and does make the component consider the field valid (showing the correct icon and not blocking saving of the form), but the underlying MDC component still sees the failed native validation and styles the input with a red border and does a little shake animation of the label when blurring the input. So I don't think we're getting around that 😞

The main reason I tried that is that I would still like mobile devices to show the appropriate keyboard for url inputs, rather than the generic keyboard. I guess we could make it possible to opt-out of the native validation for the url type (still by using `text` instead of `url` as the native type, but then at least you still get validation by default).

Thoughts?

Reply:
> We could also add a new `type`, e.g. `lime-url`, `relative-url` or something like that :thinking: 

Ah, yeah, that's a great idea! While a bit wordy, it could be `non-validated-url`…

Reply:
Hmm ok, I thought that the passed `invalid` always took precedence over anything else :thinking: Maybe it only does if you set it to `true`? :smile:

Reply:
> Maybe it only does if you set it to `true`? :smile: 

Yes, exactly 🙂

Reply:
⚡